### PR TITLE
#383 Fix counting of hackathon participants

### LINKED ISSUE #383: Fix counting of all participants
## Brief description:
Now the counting happens only in one collection, and it should be counting from both (the one for participants with team and the one for those without a team

### PR DESCRIPTION
**Pull request by IvanObreshkov**
Fix #383 
There are currently more than 120 documents in the DB. 
image

### CODE REVIEW

File: packages/api/controllers/hackathon_register_controller.go [hunk 1]
```diff
@@ -51,10 +51,11 @@ func RegisterTeamMember(c *fiber.Ctx) error {
 		ShareInfoWithSponsors:          member.ShareInfoWithSponsors,
 		WantJobOffers:                  member.WantJobOffers}
 
-	numberOfMembers, _ := GetNumberOfHackathonParticipants(ctx)
+	numberOfParticipantsWithTeam, _ := GetNumberOfHackathonParticipants(ctx)
+	numberOfParticipantsWithoutTeam, _ := GetNumberOfNoTeamParticipants(ctx)
 	numberOfTeams, _ := GetNumberOfHackathonTeams(ctx)
 
-	if numberOfMembers >= 90 {
+	if numberOfParticipantsWithTeam+numberOfParticipantsWithoutTeam >= 90 {
```
Comment (asynchroza) [inline packages/api/controllers/hackathon_register_controller.go:58]: Why don't you just extract this in one function? Either way we would always need to traverse through the two collections. 

e.g. 
```
number_of_participants = GetTotalParticipants()
```

Comment (IvanObreshkov) [reply packages/api/controllers/hackathon_register_controller.go:58]: ok

Comment (IvanObreshkov) [reply packages/api/controllers/hackathon_register_controller.go]: After the registration and probably the hackathon I would try to do a refactoring of the whole backend for the registration.

**Review by asynchroza — APPROVED** [review]
merge if all works as expected

**Pull request merged**